.bazelrc

```diff
@@ -98,3 +98,8 @@ build:clang-tidy --copt=-Wunused-private-field --copt=-Wuser-defined-warnings
 # Support user-provided user.bazelrc
 try-import %workspace%/user.bazelrc
     .
```

Comment:
As long as you're modifying this file, this errant period should be removed.  It appears to have arrived in a2568e3099 but I'm not really sure where it came from other than a typo.